safari/app-store-description.txt

```diff
@@ -0,0 +1,11 @@
+OctoLinker is a browser extension for GitHub, that turns language-specific statements like include require or import into links.
+
+OctoLinker reads the source code you're currently viewing straight from the browser or, in some cases, using the GitHub API.
+
+No source code will ever be sent to a server! Your source code stays in your browser.
+
+Based on the file type, OctoLinker invokes a plugin which then executes a regular expression on the source code to find import statements.
+
+Relative file references are resolved through the GitHub tree API in your browser. External dependencies are resolved using the OctoLinker API.
+
+Only the name of a dependency along with the registry type is sent to this service.
```

Comment:
I suggest having a single copy-pastable description that every store uses because they all have different copy now, with Chrome’s copy including some duplicate/non-clickable links.

I copied this text from the website. 

File named like https://github.com/sindresorhus/refined-github/commit/d185f27dabd85aa4b7c3f61a3f9d5ca9c6c40c1d#diff-4810e2862897d3e19bd39b745d6adae0c80afac122abf4e2aa2b845a9649fd5a

Reply:
Totally agree. I made some suggestions and once we agreed on the copy I'll update all the other store listings.